mouse_skin_obs.py

```diff
@@ -28,17 +34,29 @@ def update_cursor(self):
             scene_height = obs.obs_source_get_height(source)
             scene = obs.obs_scene_from_source(scene_source)
             scene_item = obs.obs_scene_find_source(scene, self.source_name)
+            target_item = obs.obs_scene_find_source(scene, self.target_name)
             if scene_item:
                 scale = obs.vec2()
                 obs.obs_sceneitem_get_scale(scene_item, scale)
                 scene_width, scene_height = apply_scale(
                     scale.x, scale.y, scene_width, scene_height
                 )
-                next_pos = obs.vec2()
                 next_pos.x, next_pos.y = get_position()
-                next_pos.x -= scene_width / 2
-                next_pos.y -= scene_height / 2
-                # set position to center of source where cursor is
+                next_pos.x -= self.offset_x
+                next_pos.y -= self.offset_y
+                ## base: 1920*1080, i should add something to make this automatic

```

Comment:
This is definitely a thing, base resolution directly affects performance of OBS Studio.